nupic/research/frameworks/pytorch/imagenet/imagenet_experiment.py

```diff
@@ -869,47 +894,173 @@ def get_free_port(self):
 
     @classmethod
     def get_execution_order(cls):
+        exp = "SupervisedExperiment"
```

Comment:
Why are you introducing this constant and the extra string concatenations?

Reply:
This was an idea by @mvacaporale to make the code look cleaner, there is no reason to repeat "ImagenetExperiment" dozens of times. It also helps avoid errors from typos. I see no harm in this.

Reply:
It does not look cleaner to me and there is a performance cost of 25+ extra string concatenation

Reply:
It's cleaner in terms of non-wrapping lines. And I think the performance cost is small, right?

I'm fine either way though.

Reply:
If wrapping lines is the issue, I would `# noqa: E501` to the method before adding extra string concatenations

Reply:
Style can be argued. But the efficiency argument doesn't hold. This is O(1) and only executed once at initialization, we are talking about nanoseconds here. 

Given the differences in opinion, I will abide by the opinion of whoever coded in the first place. @mrcslws git blame says it is your code. If you are not against the changes, I will keep them.